src/index.html

```diff
@@ -22,6 +22,347 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div>
+      <header class="header">
+        <a href="#">
+          <img
+            class="img"
+            src="/src/images/logo.png"
+            alt="MOYO logo"
+          />
+        </a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                class="nav__link is-active"
```

Comment:
The color of the navigation links should be `#060b35` instead of the current color. This needs to be updated in the CSS to meet the task requirements.